src/libravi/RaviSession.ts

```diff
@@ -22,6 +22,25 @@ export enum RaviSessionStates {
   CLOSED = "closed"
 };
 
+export interface RaviSessionParams {
+  /**
+   * The minimum jitter buffer duration.
+   * 
+   * The lower the jitter buffer duration, the lower the round-trip audio latency.
+   * However: set the jitter buffer duration too low on an unreliable connection, and users receiving an audio stream may hear interruptions or
+   * "drop-outs" in the audio stream. Set the jitter buffer duration high to reduce the possibility of dropouts at the cost of higher round-trip audio latency.
+   */
+  audioMinJitterBufferDuration?: number;
+  /**
+   * The maximum jitter buffer duration.
+   * 
+   * The lower the jitter buffer duration, the lower the round-trip audio latency.
+   * However: set the jitter buffer duration too low on an unreliable connection, and users receiving an audio stream may hear interruptions or
+   * "drop-outs" in the audio stream. Set the jitter buffer duration high to reduce the possibility of dropouts at the cost of higher round-trip audio latency.
```

Comment:
"Set the jitter buffer duration high to reduce the possibility of dropouts at the cost of potentially higher round-trip audio latency on poor connections."